fiftyone/core/frame.py

```diff
@@ -361,7 +383,9 @@ class Frame(Document):
     """
 
     # Instance references keyed by [collection_name][sample_id][frame_number]
-    _instances = defaultdict(lambda: defaultdict(weakref.WeakValueDictionary))
+    _instances = defaultdict(
```

Comment:
@benjaminpkane doesn't this break the methods like `_rename_fields()` below? They don't index by `view`.

`SampleView`s aren't singletons, so can the behavior in `from_doc` below be to always create a new `Frame` instance if the `Frame` is being loaded in a view?